src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/CodeReaderWriter/CodeModifierConfig.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.DotNet.MSIdentity.CodeReaderWriter
+{
+    public class CodeChange
```

Comment:
we usually have all classes in individual files, either way is fine , when you follow the same pattern - in this repo i still see individual files per class...